extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -212,6 +237,37 @@ export const SignTransaction = () => {
     );
   }
 
+  const hasLoadedBalances =
+    accountBalanceStatus !== ActionStatus.PENDING &&
+    accountBalanceStatus !== ActionStatus.IDLE;
+
+  if (!hasLoadedBalances) {
+    return <Loading />;
+  }
+
+  const hasBalance =
+    hasLoadedBalances && accountBalanceStatus !== ActionStatus.ERROR;
+  const hasEnoughXlm = accountBalances.balances?.native.available.gt(
+    stroopToXlm(_fee as string),
+  );
+  if (hasBalance && currentAccount.publicKey && !hasEnoughXlm) {
+    return (
```

Comment:
hm, i don't think we should block signing. I think maybe we should just warn, and if the user still wants to sign, they can. 

Thinking of a scenario where a different account will be submitting and therefore will be responsible for the fee

Reply:
hmm yeah good call, let me see if I can make this non blocking.

Reply:
@piyalbasu what do you think of this? Made the warning dismissable in c1cd38e8ad7a97d0a4faa37a654804d1d1cba4b5